examples/GainPlugin/jsui/src/App.js

```diff
@@ -3,11 +3,12 @@ import Label from './Label';
 import Meter from './Meter';
 import React, { Component } from 'react';
 import Slider from './Slider';
+import ParameterToggleButton from './ParameterToggleButton'
 import {
   Canvas,
   Image,
-  View,
   Text,
+  View
```

Comment:
1000% nit pick, but I prefer trailing commas everywhere in javascript. The reason is simple: trailing commas aren't a problem in js, and here for example with one import name per line, adding a _new_ import after View would involve touching the `View` line as well to add the necessary comma. That means touching 2 lines where you only needed 1, which makes the git diff slightly more complicated. For longer lines, this effect gets more impactful.

Again, supreme nitpick, but that's why you'll see me write trailing commas everywhere :)